config.py

```diff
@@ -242,6 +242,10 @@ class Production(Live):
         'send-daily-seller-email': {
             'task': 'app.tasks.mailchimp.send_new_briefs_email',
             'schedule': crontab(hour=7, minute=0)
+        },
+        'process_closed_briefs': {
+            'task': 'app.tasks.brief_tasks.process_closed_briefs',
+            'schedule': crontab(hour=2)
```

Comment:
As the schedule is in UTC timezone, this is 2AM UTC, i.e. 12PM AEST. Did you mean 14? this would be midnight AEST. Or if you meant 2AM AEST, then this will need to be 16 for 4PM UTC.